### DISCUSSION TITLE
Extensions: Where can I find docs for keywords used in Scheme files?

### DISCUSSION THREAD

**Discussion by marcospb19** [zed-industries/zed#21833] [Feature Requests]
While reading docs about extensions and languages, I found this:

https://github.com/zed-industries/zed/blob/28faba12a2799ef535aaf3158f7437f70ff523c8/docs/src/extensions/languages.md?plain=1#L307-L312

As well as this:
https://github.com/zed-industries/zed/blob/28faba12a2799ef535aaf3158f7437f70ff523c8/docs/src/extensions/languages.md?plain=1#L286-L292

Are `pair`, `array` and `_` special keywords?

If so, where can I read more about these keywords?

---

My first thought was: the first symbol is a name for what appears next, however, with:

```scheme
 (array (number) @redact) 
 (array (string) @redact) 
```
 
It doesn't make sense to me that an array is defined as a number :thinking:.

---

What about these?
https://github.com/zed-industries/zed/blob/62a6a755eceae895b2f46763b3200959dbde5218/docs/src/extensions/languages.md?plain=1#L343-L344

I know it's to-be-documented

https://github.com/zed-industries/zed/blob/62a6a755eceae895b2f46763b3200959dbde5218/docs/src/extensions/languages.md?plain=1#L359

But I couldn't find any reference in the project's source code.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
